### ISSUE TITLE
[BUG] GitOps configuration error

### ISSUE THREAD

**Issue by irtiash** [microsoft/NubesGen#437]
**Describe the bug**
(1/10) Checking if the project is configured on GitHub...
The project isn't configured on GitHub, GitOps configuration is disabled.

**To Reproduce**
.\nubesgen-cli-windows.exe gitops

**Expected behavior**

**Screenshots**
![image](https://github.com/microsoft/NubesGen/assets/64924545/c4aacbea-65e6-4fcc-9722-4d426f4cecd1)

**Comment by jdubois** [comment]
Indeed, I didn't think of this, and the message needs to be more clear and point to the documentation (there's not much more to do than better documentation in this use case, if you don't have GitHub then you can't have the GitOps option).

**Comment by irtiash** [comment]
But github is present and it is all setup but still get this error

**Comment by jdubois** [comment]
Oh sorry, I see you have edited the bug report so my initial comment isn't valid anymore.
As you have GitHub installed and configured, that's another issue:  could you fork https://github.com/jhipster/jhipster-sample-app and then test the Nubesgen CLI on your fork? Maybe there's something unusual with the repo you are configuring.

**Comment by irtiash** [comment]
Ok, I'll check it out. Thanks
